common.sh

```diff
@@ -56,9 +56,18 @@ export KNI_INSTALL_FROM_GIT=${KNI_INSTALL_FROM_GIT:-}
 # See https://openshift-release.svc.ci.openshift.org for release details
 #
 # if we provide OPENSHIFT_RELEASE_IMAGE, do not curl. This is needed for offline installs
+if echo $SHELLOPT | grep -q nounset; then
+    _reset_nounset=true
+else
+    _reset_nounset=false
+fi
+set +u
 if [ -z "${OPENSHIFT_RELEASE_IMAGE}" ]; then
```

Comment:
Wouldn't it be simpler to change this to:

   if [ -z "${OPENSHIFT_RELEASE_IMAGE:-}" ]; then

?

Reply:
Where's the fun in that?

I'll wait for my cluster to build and push the change.